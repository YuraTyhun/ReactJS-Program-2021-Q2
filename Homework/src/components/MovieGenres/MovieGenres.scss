.movie-genres {
    padding: 0 2px;
    font-size: 15px;
    letter-spacing: 0.7px;
}