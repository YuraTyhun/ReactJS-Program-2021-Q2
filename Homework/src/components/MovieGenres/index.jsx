import React from 'react';
import PropTypes from 'prop-types';

import './MovieGenres.scss';

const MovieGenres = ({ genres }) => (
    <div className="movie-genres">{genres.join(', ')}</div>
);

MovieGenres.propTypes = {
    genres: PropTypes.arrayOf(PropTypes.string).isRequired
}

export default MovieGenres;