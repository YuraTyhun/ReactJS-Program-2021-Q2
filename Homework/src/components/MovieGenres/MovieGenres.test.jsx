import React from 'react';
import renderer from 'react-test-renderer';

import MovieGenres from './index';

it('Genres render correctly', () => {
  const mockGenres = ['Comedy', 'Action', 'Horror'];
  const genres = renderer
    .create(<MovieGenres genres={mockGenres} />)
    .toJSON();
  expect(genres).toMatchSnapshot();
});
