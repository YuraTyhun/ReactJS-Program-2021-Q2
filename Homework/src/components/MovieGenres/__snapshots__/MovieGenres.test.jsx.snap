// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genres render correctly 1`] = `
<div
  className="movie-genres"
>
  Comedy, Action, Horror
</div>
`;
